front_end/panels/ai_chat/LLM/AnthropicProvider.ts

```
@@ -0,0 +1,469 @@
+        }
+        return { type: 'text', text: String(item) };
+      });
+    }
+
+    return [{ type: 'text', text: String(content) }];
+  }
+
+  /**
+   * Converts LLMMessage format to Anthropic Messages API format
+   */
+  private convertMessagesToAnthropic(messages: LLMMessage[]): { system?: string, messages: any[] } {
```

ESLint

[error] Optional property 'system' should be defined after required properties.

(rulesdir/enforce-optional-properties-last)